commit d9c6fa684883a86a205a069be385ed2e87ac29a2 (refs/changes/94/4594/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2018-07-27T18:46:16+00:00 (1 year, 2 months ago)
    
    OS-7097 ahciem does not build with newer gcc
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/usr/src/cmd/ahciem/ahciem.c b/usr/src/cmd/ahciem/ahciem.c
index 65ecf2a425..3c3a473f51 100644
--- a/usr/src/cmd/ahciem/ahciem.c
+++ b/usr/src/cmd/ahciem/ahciem.c
@@ -65,7 +65,7 @@ ahciem_usage(const char *fmt, ...)
 }
 
 static const char *
-ahciem_led_to_string(ahci_em_led_state_t led)
+ahciem_led_to_string(uint_t led)
 {
 	switch (led) {
 	case AHCI_EM_LED_IDENT_ENABLE:
